include/deal.II/base/data_out_base.h

```diff
@@ -872,29 +847,15 @@ namespace DataOutBase
     const char *zone_name;
 
     /**
-     * Constructor
+     * Constructor.
      */
     TecplotFlags (const char *tecplot_binary_file_name = NULL,
                   const char *zone_name = NULL);
 
     /**
-     * Declare all flags with name and type as offered by this class, for use
-     * in input files.
-     */
-    static void declare_parameters (ParameterHandler &prm);
-
-    /**
-     * Read the parameters declared in declare_parameters() and set the flags
-     * for this output format accordingly.
-     *
-     * The flags thus obtained overwrite all previous contents of this object.
-     */
-    void parse_parameters (const ParameterHandler &prm) const;
-
-    /**
-     * Return an estimate for the memory consumption, in bytes, of this
-     * object. This is not exact (but will usually be close) because calculating
-     * the memory usage of trees (e.g., <tt>std::map</tt>) is difficult.
+     * Return an estimate for the amount of memory used by this object. This
+     * will be an underestimate if either of the two <tt>const char *</tt>s
+     * point to a buffer longer than its contained string.
```

Comment:
I think the last sentence is too technical. Let's just drop it -- the first sentence already says that it's an estimate.